File: BeerReal/app/src/main/java/fr/epita/beerreal/ui/home/FeedItem.java
package fr.epita.beerreal.ui.home;

import fr.epita.beerreal.csv.Line;

public class FeedItem {
    private String imageUrl;  // or local path, URI etc.
    private String feedText;
    private Line line;

    public FeedItem(String imageUrl, String feedText, Line line) {
        this.imageUrl = imageUrl;
        this.feedText = feedText;
        this.line = line;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public Line getLine() {
        return line;
    }
}

File: v3/www/js/home/feedItem.js
import '../../css/home/beerCard.css';

export class FeedItem {
    constructor(imageUrl, line) {
        this.imageUrl = imageUrl;
        this.line     = line;
    }

    getImageUrl() { return this.imageUrl; }
    getLine()     { return this.line; }
}